integrationtest/mapper/pojo-base/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/IndexedEmbeddedBaseIT.java

```diff
@@ -1617,7 +1822,480 @@ class EntityC {
+						.typeContext( Model.IndexedEntity.class.getName() )
+						.pathContext( ".included<no value extractors>.subIncluded<no value extractors>.subSubIncluded" )
+						.failure(
+								"Found conflicting include/exclude paths configuration.",
+								"Included paths are: '[]', excluded paths are: '[subSubIncludedString]'.",
+								"To resolve the conflict either change the configuration of the current property or look at `includePaths` or `excludePaths` of the first node of paths in '[included.subIncluded.subSubIncluded.subSubIncludedString]'."
+						)
+				);
+	}
+
+	@Test
+	public void parentExcludeChildInclude() {
+		class Model {
+			@Indexed(index = INDEX_NAME)
+			class IndexedEntity {
+				@DocumentId
+				Integer id;
+				@KeywordField
+				String indexedEntityString;
+
+				@IndexedEmbedded(excludePaths = "subIncluded.subSubIncluded.subSubIncludedString")
+				Included included;
+
+				public IndexedEntity(Integer id, String indexedEntityString, Included included) {
+					this.id = id;
+					this.indexedEntityString = indexedEntityString;
+					this.included = included;
+				}
+			}
+
+			class Included {
+				@KeywordField
+				String includedString;
+
+				@IndexedEmbedded
+				SubIncluded subIncluded;
+
+				public Included(String includedString, SubIncluded subIncluded) {
+					this.includedString = includedString;
+					this.subIncluded = subIncluded;
+				}
+			}
+
+			class SubIncluded {
+				@KeywordField
+				String subIncludedString;
+
+				@IndexedEmbedded(includePaths = "subSubIncludedString")
+				SubSubIncluded subSubIncluded;
+
+				public SubIncluded(String subIncludedString) {
+					this.subIncludedString = subIncludedString;
+				}
+			}
+
+			class SubSubIncluded {
+				@KeywordField
+				String subSubIncludedString;
+
+				public SubSubIncluded(String subSubIncludedString) {
+					this.subSubIncludedString = subSubIncludedString;
+				}
+			}
+		}
+
+		assertThatThrownBy( () -> setupHelper.start()
+				.withAnnotatedEntityTypes( Model.IndexedEntity.class )
+				.setup() )
+				.isInstanceOf( SearchException.class )
+				.satisfies( FailureReportUtils.hasFailureReport()
+						.typeContext( Model.IndexedEntity.class.getName() )
+						.pathContext( ".included<no value extractors>.subIncluded<no value extractors>.subSubIncluded" )
+						.failure(
+								"Found conflicting include/exclude paths configuration.",
+								"Included paths are: '[subSubIncludedString]', excluded paths are: '[]'.",
+								"To resolve the conflict either change the configuration of the current property or look at `includePaths` or `excludePaths` of the first node of paths in '[included.subIncluded.subSubIncluded.subSubIncludedString]'."
+						)
+				);
+	}
+
+	/*
+	 * Since parent adds a path to be excluded and child uses an include filter that explicitly includes a different field -- there's no conflict.
+	 * TODO: filter may look as redundant, do we want to fail in such case?
```

Comment:
Yes we do want to fail. `excludePaths = "subIncluded.subSubIncluded.subSubIncludedString"` is effectively what the "useless exclude paths" detection is about: this filter should be reported as useless.